builder/createStatesJs.js

```diff
@@ -42,11 +42,11 @@ const statesObject = {
   ...readAdditionalStates(),
 };
 
-let result = [];
+const result = [];
 result.push("'use strict';");
 result.push(`const states = ${JSON.stringify(statesObject, null, 2)};`);
 result.push('Object.freeze(states);');
-result.push('module.exports = states;');
+result.push('export default states;');
 result.push('');
 
-fs.writeFileSync(`${__dirname}/../src/generated/states.js`, result.join('\n'));
```

Comment:
I think you forgot to rerun the script after changing it

Reply:
i did and nothing changed cuz i migrate states.ts manually before

Reply:
how do you explain that all lines have changed in the new version (quotes)?

Reply:
because we have now a new file states.ts

Reply:
before:
![image](https://user-images.githubusercontent.com/2352663/210443140-5d282a98-c084-4bc1-a203-970d5b408ca8.png)

after:
![image](https://user-images.githubusercontent.com/2352663/210443175-2aca293b-098d-4308-a636-0d06c641614e.png)

The style of the strings should not have changed (in this script, I still see JSON.stringify, which outputs double quotes)

Reply:
I used prettier after generating the file

Reply:
I see. Please keep it as-is. Regenerating the file shouldn't end up in uncommitted changes (unless the data has changed)

Reply:
okay